pkg/commands/app.go

```diff
@@ -304,6 +305,7 @@ func NewFilesystemCommand(globalFlags *flag.GlobalFlagGroup) *cobra.Command {
 		MisconfFlagGroup:       flag.NewMisconfFlagGroup(),
 		ModuleFlagGroup:        flag.NewModuleFlagGroup(),
 		RemoteFlagGroup:        flag.NewClientFlags(), // for client/server mode
+		RegistryFlagGroup:      flag.NewRegistryFlagGroup(),
```

Comment:
It looks like we are not using credentials in `fs` mode, or am I missing something?

Reply:
The credential will be used for downloading DB. I'll raise another PR.